third-party/chpl-venv/c2chapel-requirements.txt

```diff
@@ -1,2 +1,2 @@
 # tools/c2chapel/Makefile should match so fakeHeaders download matches
```

Comment:
as the comment says you should update the version in tools/c2chapel/Makefile

Reply:
thank you for the review.  I've updated the version.